tools/README.md

```diff
@@ -1,6 +1,31 @@
 AMS data tools
 ========================
 
+
+clear_kafka_topics
+------------------
+
+Clear kafka topics is a script that deletes all kafka topics that
+are not present in AMS.
```

Comment:
deletes all kafka orphan topics. This means that the topics are not used anymore.